docs/content/rn-fixed-issues-ref.adoc

```diff
@@ -0,0 +1,11 @@
+
+[id='rn-fixed-issues-ref']
+
+= Fixed Issues
+
+The following issues are resolved in {PRODUCT} {PATCH_VERSION_NO}:
+
+
+* [INTLY-2118] - AMQ Online serviceinstance dashboard URLs are not updated from 1.0 to 1.1
```

Comment:
serviceinstance = service instance (note the space)?

Reply:
tis ok